continuous_integration/travis/install.sh

```diff
@@ -39,17 +48,17 @@ conda install -c conda-forge -q \
     ipywidgets \
     joblib \
     jupyter_client \
-    msgpack-python>=0.6.0 \
+    msgpack-python \
     netcdf4 \
     paramiko \
     prometheus_client \
     psutil \
-    pytest>=4 \
+    pytest \
     pytest-timeout \
-    python=$PYTHON \
     requests \
+    scikit-learn \
     scipy \
-    tblib>=1.5.0 \
+    tblib \
```

Comment:
Why remove these version pinnings?  I'm not disagreeing with the action, I'm just curious.

Reply:
They were being interpreted by bash as stdout redirection, creating files such as ``=1.5.0``. They were never received by conda to begin with.

I don't think minimum versions should be pinned in conda files because conda will already install the very latest available version anyway (save conflicts); the only exception I can think of is when you have ``channels: [pkgs/main, conda-forge]``; in such a case you may want to explicitly request a version that is in conda-forge but not in pkg/main yet.

Reply:
I have anecdotally observed the conda solver failing to grab the latest version. If a minimum compatible version is known, it can be helpful to document that in this way. Would putting quotes around the full expression resolve the bash issue?

Reply:
done